cli/src/config/colors.toml

```diff
@@ -78,9 +78,11 @@
 "diff binary" = "cyan"
 "diff file_header" = { bold = true }
 "diff hunk_header" = "cyan"
-"diff removed" = { fg = "red", underline = true }
-"diff added" = { fg = "green", underline = true }
-"diff line_number" = { underline = false }
+"diff removed" = { fg = "red" }
+"diff added" = { fg = "green" }
+"diff color_words added" = { underline = true }
+"diff color_words removed" = { underline = true }
```

Comment:
I would use `diff added token` or something so that we can reuse the style if we add word-diff support in git diff.

Reply:
Good idea. I've sent #3974 for that. Maybe I should have updated this one instead, not sure. It's completely different code...